src/swift/DispatchStubs.cc

```diff
@@ -168,12 +174,24 @@ static void _dispatch_overlay_constructor() {
 
 SOURCE(DATA_ADD)
 SOURCE(DATA_OR)
+#if HAVE_MACH
 SOURCE(MACH_SEND)
 SOURCE(MACH_RECV)
 SOURCE(MEMORYPRESSURE)
+#endif
 SOURCE(PROC)
 SOURCE(READ)
 SOURCE(SIGNAL)
 SOURCE(TIMER)
 SOURCE(VNODE)
```

Comment:
vnode should be ignored too unless libkqueue support is good (but I don't think so)